PyInstaller/makespec.py

```diff
@@ -376,6 +386,9 @@ def main(scripts, name=None, onefile=False,
     else:
         cipher_init = cipher_absent_template
 
+    if background:
+        console = False
```

Comment:
IMHO this is no a good idea. This would disable using the same spec-file for both windows (where one may want console=True) and OS X (where on may want to use background=True). And because background==True has higher precedence then console (see first chunk) it is not necessary to set console to false here.